pom.xml

```diff
@@ -68,35 +68,6 @@
       <version>${kafka.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>kafka_2.12</artifactId>
-      <version>${kafka.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.scala-lang</groupId>
-          <artifactId>scala-library</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-
-    <!-- Kafka requires these dependencies: declare this dependency to force vertx-kafka-client to use this one. These are the versions used by vert.x -->
```

Comment:
@vietj are you ok removing this part as well?

Reply:
I'm not

Reply:
@vietj can you elaborate?  As far as I can see this dependency is not used any more since 3ac632f.

Reply:
which dependency are we talking about ?

Reply:
ok so to clarify, I'm fine removing unused dependencies that are not used somewhere else in the stack like scala-library, however I am not removing dependencies like slf4j that force the version

Reply:
Agreed, missed that.

I fixed the version of slf4j-api again.